docs/package.json

```diff
@@ -26,12 +26,12 @@
     "next": "14.2.12",
     "nextra": "2.13.4",
     "nextra-theme-docs": "2.13.4",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
   },
   "devDependencies": {
-    "@types/node": "^20.8.10",
+    "@types/node": "20.12.12",
     "eslint-config-next": "13.5.6",
-    "typescript": "^5.2.2"
```

Comment:
it's a version bump here

Reply:
This is because when you run `pnpm install` at the root of the repo with the old rule in the package json (^5.2.2) version `5.4.3` is what you end up having installed. If you want to check, on @develop do

```
pnpm i 
cd ./docs/
pnpm list typescript # => 5.4.3
```

Pinning to the real installed version is less disruptive than bumping down a version IMO